versions/v1/units.json

```
@@ -6412,6 +6412,92 @@
     "source": "Experimental Methods and Instrumentation for Chemical Engineers - ISBN 9780444538055 - https://isbnsearch.org/isbn/9780444538055",
     "sourceReference": null
   },
+  {
+    "externalId": "pressure_gradient:pa-per-m",
+    "name": "PA-PER-M",
+    "quantity": "Pressure Gradient",
+    "longName": "Pressure Differential per metre in Pa/m",
+    "aliasNames": [
+      "pascal per metre",
+      "Pascal per metre",
+      "pa per m",
+      "Pa/m",
+      "pa/m",
+      "Pa / m",
+      "pa / m"
+    ],
+    "symbol": "Pa/m",
+    "conversion": {
+      "multiplier": 1.0,
+      "offset": 0.0
+    },
+    "source": "qudt.org",
+    "sourceReference": "https://qudt.org/vocab/quantitykind/PressureGradient"
```

Here we link to **qudt.org** `units` and not `quantityKinds`. In this case, I can see the base unit is missing in **qudt.org**, but we could define it as a custom unit based on an entry of the **qudt.org** catalog. If you check on unique value of source in the unit catalog, you'll see that there other cases where we adopted the same approach.
```suggestion
    "source": "Custom based on qudt.org - https://qudt.org/vocab/unit/HectoPA-PER-M",
    "sourceReference": null
```